scripts/debug_tools/failure_lib.py

```diff
@@ -0,0 +1,53 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+import json
+import os
+
+
+def findPathEnd(string, path_begin):
+    """ Find one path in string.
```

Comment:
Please format the docstrings either

```py
"""This way."""
```

or 

```
"""
This way.
"""
```

I vote in favor of the second version.